core/main/src/processor/main_context_processor.rs

```diff
@@ -92,28 +79,7 @@ impl MainContextProcessor {
         {
             if let Some(session) = response.payload.extract() {
                 state.session_state.insert_account_session(session);
-                MetricsState::update_account_session(state).await;
                 event = CapEvent::OnAvailable;
-                let state_c = state.clone();
-                // update ripple context for token asynchronously
-                tokio::spawn(async move {
```

Comment:
Guessing we're now doing this wherever `MetricsState` got moved to?

Reply:
Yes its moved to distributor extension as session manager